.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "wasmCloud",
+	"image": "rust:bullseye",
+	"features": {
+		"devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
+		"ghcr.io/lee-orr/rusty-dev-containers/cargo-binstall:latest": {},
+		"ghcr.io/lee-orr/rusty-dev-containers/cargo-watch:latest": {},
+		"ghcr.io/lee-orr/rusty-dev-containers/cargo-nextest:latest": {},
+		"ghcr.io/devcontainers/features/nix": {}
```

Comment:
If we're adding this, does it support flakes out-of-the-box? Where can I find documentation around this?